### DISCUSSION TITLE
persisting data during task

### DISCUSSION THREAD

**Discussion by pmavros** [jspsych/jsPsych#2503] [Q&A]
Hi everyone, 

Second question for the day, I am not sure how this is called correctly:

In some survey systems (e.g. qualtrics) you don't necessarily loose your session if you happen to reload the webpage before finishing the survey. The data are also pushed frequenly to the database (this leads to partial surveys, but that's another issue). 

Is there a way to achieve this with jsPsych?

Thank you, 
Panos

**Comment by jodeleeuw** [comment]
Hi @pmavros,

At the moment, there isn't a way to do this. It it something we've thought about a little bit, but would require a good amount of work to complete and isn't high on the priority list for now.
